crates/ruff/src/settings/pyproject.rs

```diff
@@ -225,8 +225,8 @@ ignore = ["E501"]
             pyproject.tool,
             Some(Tools {
                 ruff: Some(Options {
-                    extend_select: Some(vec![RuleCodePrefix::RUF100.into()]),
-                    ignore: Some(vec![RuleCodePrefix::E501.into()]),
+                    extend_select: Some(vec![codes::Ruff::_100.into()]),
```

Comment:
So this is a result of the prefix now being encoded via `codes::Ruff`, and so the identifier itself starts with a number, and so we need the underscore. Is that correct?

Reply:
Yes, see `get_prefix_ident` in `ruff_macros/src/rule_code_prefix.rs`:

> // Identifiers in Rust may not start with a number.